src/Options.php

```diff
@@ -848,6 +848,9 @@ public function getDpi()
      */
     public function setFontCache($fontCache)
     {
+        if (!is_dir($fontCache) && !mkdir($fontCache)) {
+            trigger_error("Unable to create fontCache - a valid value should be specified.", E_USER_WARNING);
```

Comment:
Thinking Dompdf should return without setting the option if it's unable to create the directory.

Reply:
Sometimes it's hard to balance between just making it work and asking developers to do their job 😅. You're probably right that a lot of users will miss the warnings. Is your suggestion to not make the directory but add the warning, then in a future major release add the mkdir functionality?